### PR TITLE
added lesson02

### PR DESCRIPTION
**Pull request by SvetaPiven**

### CODE REVIEW

File: lesson02/src/task1.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        y = in.nextInt();
+        in.close();
+        System.out.println(x/y);
+        System.out.println(x%y);
+    }
+}
+
+
+
+
```
Comment (rakovets) [inline lesson02/src/task1.java:19]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: lesson02/src/task1.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        Scanner in = new Scanner(System.in);
+        int x;
+        int y;
+        System.out.print("Input x: ");
+        x = in.nextInt();
+        System.out.print("Input y: ");
+        y = in.nextInt();
+        in.close();
+        System.out.println(x/y);
+        System.out.println(x%y);
```
Comment (rakovets) [inline lesson02/src/task1.java:13]: "Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"

File: lesson02/src/task2.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        System.out.print("Input width: ");
+        width = in.nextInt();
+        System.out.print("Input height: ");
+        height = in.nextInt();
+        in.close();
+        System.out.println("Area: " + width * height);
+        System.out.println("Perimeter: " + (width + height) * 2);
+    }
+}
+
```
Comment (rakovets) [inline lesson02/src/task2.java:16]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: lesson02/src/task3.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+        int radius;
+        final double Pi = 3.1415926536;
+        System.out.print("Input radius: ");
+        radius = in.nextInt();
+        in.close();
+        System.out.println("Area: " + Pi * radius * radius);
+        System.out.println("Circumference: " + 2 * Pi * radius);
+    }
+}
+
```
Comment (rakovets) [inline lesson02/src/task3.java:14]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: lesson02/src/task4.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+public class task4 {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        int number;
+        System.out.print("Input number: ");
+        number = in.nextInt();
+        in.close();
+        int x = number / 100;
+        int y = number % 10;
+        int z = (number / 10) %10;
```
Comment (rakovets) [inline lesson02/src/task4.java:11]: "Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"

File: lesson02/src/task5.java [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.println("Input x: ");
+        double x = in.nextDouble();
+            if (x % 1 == 0) {
+                System.out.println("Whole");
+             } else {
+                System.out.println("No whole");
+             }
+    }
```
Comment (rakovets) [inline lesson02/src/task5.java:12]: "Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"

File: lesson02/src/task5.java [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+}
+
```
Comment (rakovets) [inline lesson02/src/task5.java:14]: "Violation: Useless blank lines
Solution: Remove blank lines"

**Review by rakovets — APPROVED** [review]

File: lesson02/src/task1.java (hunk 1 above)
Comment (SvetaPiven) [reply lesson02/src/task1.java]: Task1 fixed

File: lesson02/src/task1.java (hunk 2 above)
Comment (SvetaPiven) [reply lesson02/src/task1.java]: Task2 fixed

File: lesson02/src/task2.java (hunk 3 above)
Comment (SvetaPiven) [reply lesson02/src/task2.java]: Task2fixed

File: lesson02/src/task3.java (hunk 4 above)
Comment (SvetaPiven) [reply lesson02/src/task3.java]: Task3 fixed

File: lesson02/src/task4.java (hunk 5 above)
Comment (SvetaPiven) [reply lesson02/src/task4.java]: Task4 fixed

File: lesson02/src/task5.java (hunk 6 above)
Comment (SvetaPiven) [reply lesson02/src/task5.java]: Task5 fixed

File: lesson02/src/task5.java (hunk 7 above)
Comment (SvetaPiven) [reply lesson02/src/task5.java]: Task5 fixed

**Pull request merged**